Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -144,11 +148,13 @@ void InterfacePane::CreateUI()
   m_checkbox_top_window = new QCheckBox(tr("Keep Window on Top"));
   m_checkbox_use_builtin_title_database = new QCheckBox(tr("Use Built-In Database of Game Names"));
   m_checkbox_use_userstyle = new QCheckBox(tr("Use Custom User Style"));
+  m_checkbox_use_covers = new QCheckBox(tr("Use Game Covers in Grid Mode"));
```

Comment:
It would be useful to have some kind of note that the covers get downloaded from the internet (in case someone is on a metered connection). Maybe something like "Download Game Covers for Use in Grid Mode"?